.github/workflows/CI.yml

```diff
@@ -60,8 +62,14 @@ jobs:
         }
         Write-Host "Runner started"
         $url = "$runner_host/run?notimers=true&$env:JQUERY&constellation=$env:CONSTEL"
-        & "C:\Program Files\Internet Explorer\iexplore.exe" "$url"
-        Write-Host "IE started"
+        $browser = $env:BROWSER
+        $browserPaths = @{
+          IE = "C:\Program Files\Internet Explorer\iexplore.exe";
+          MSEdge = "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe";
+        }
+        Write-Host "Starting $browser with $url"
```

Comment:
```suggestion
        Write-Host "Starting $browserPaths[$browser] with $url"
```

Reply:
https://github.com/DevExpress/DevExtreme/runs/1309891106?check_suite_focus=true#step:6:45

Reply:
Done in cbc21ec.